bigbluebutton-web/grails-app/conf/bigbluebutton.properties

```diff
@@ -91,6 +91,16 @@ numPluginManifestsFetchingThreads=5
 #------------------------------------
 pluginManifestFetchTimeout=15
 
+#------------------------------------
+# Each plugin manifest source URL fetch shouldn't take longer than this time (seconds)
+#------------------------------------
+pluginManifestsFetchUrlResponseTimeout=15
```

Comment:
Please add to https://github.com/bigbluebutton/bigbluebutton/blob/v3.0.7/docs/docs/new-features.md?plain=1#L352-L355

Reply:
Done